### PR TITLE
part 8 for Client Side Javascript

### PR DESCRIPTION
**Pull request by girlinitworld**
Main files are index.html and script.js (file somePage.html is only for task 2 purpose)

### CODE REVIEW

File: part8-clientSide/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      <p>Paragraph</p>
+      <section>Not a Paragraph</section>
+      <p>Paragraph</p>
+      <section>Not a Paragraph</section>
+    </div>
+
+    <div>
+      <h1>Task 8 - <small>Replace paragraphs text</small></h1>
+      <button onclick="replaceParagraphText()">Replace paragraphs text</button>
+      <p id="replaceText">Paragraph</p>
```
Comment (matthewInforma) [inline part8-clientSide/index.html]: You should have **only one unique** `id` per page. Please replace replaceText by a `class`.

File: part8-clientSide/script.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  let paragraphs = document.querySelectorAll("p");
+
+  for (let i = 0; i < paragraphs.length; i++) {
+    paragraphs[i].style.backgroundColor = "yellow";
+  }
+}
+//===================================================================================================================================================================
+// 8. Replace the text content of a paragraph with a new value when a button is clicked.
+function replaceParagraphText() {
+  let paragraphs = document.querySelectorAll("p#replaceText");
```
Comment (matthewInforma) [inline part8-clientSide/script.js]: let paragraphs = document.querySelectorAll("p#replaceText");

Please replace let to const. You want fetch p with a class replaceText.

File: part8-clientSide/index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    <div>
+      <h1>Task 9 - <small>customize element</small></h1>
+      <button onclick="customizeElement()">Customize Element</button>
+      <div id="myElement">ELEMENT</div>
+    </div>
+
+    <div>
+      <h1>Task 10 - <small>trigger two different alerts</small></h1>
+      <button id="myButton">Click</button>
```
Comment (matthewInforma) [inline part8-clientSide/index.html]: in order to see event propagation you should move button inside the parent element, like so:

```html
<div id="parentElement">
    <button id="myButton">Click</button>
    <p>Parent Element</p>
</div>
```
Then try to click button. You will see in which order alerts are triggered.

**Review by matthewInforma — APPROVED** [review]

File: part8-clientSide/script.js (hunk 2 above)
Comment (girlinitworld) [reply part8-clientSide/script.js]: why const in this case?

Comment (matthewInforma) [reply part8-clientSide/script.js]: it can be either let and const, but in most cases when I query the DOM elements I keep it in const.

**Pull request merged**